### PR TITLE
Updated the privileges for backup and restore + externalIO implicit access privilege

### PR DESCRIPTION
**Pull request by kathancox**
Fixes [DOC-5528](https://cockroachlabs.atlassian.net/browse/DOC-5528), [DOC-5505](https://cockroachlabs.atlassian.net/browse/DOC-5505), [DOC-5558](https://cockroachlabs.atlassian.net/browse/DOC-5558), [DOC-4781](https://cockroachlabs.atlassian.net/browse/DOC-4781)

Updated the privileges section for new 22.2 privilege model on the following pages:

- `BACKUP` 
- `RESTORE`
- `CREATE SCHEDULE FOR BACKUP` added more detail here and copied over the `BACKUP` privs.

Also added the `EXTERNALIOIMPLICITACCESS` privilege to the following pages:

- `BACKUP`
- `RESTORE`
- `IMPORT`
- `IMPORT INTO`
- `EXPORT`

**Note**: changefeed privilege work will be a separate PR, including anything related to `EXTERNALIOIMPLICITACCESS` and changefeeds.

### CODE REVIEW

**Comment by kathancox** [comment]
Backup page: https://deploy-preview-15043--cockroachdb-docs.netlify.app/docs/dev/backup.html#required-privileges
Restore page: https://deploy-preview-15043--cockroachdb-docs.netlify.app/docs/dev/restore.html
Create schedule: https://deploy-preview-15043--cockroachdb-docs.netlify.app/docs/dev/create-schedule-for-backup.html#required-privileges

File: _includes/v22.2/backups/destination-privileges.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+The user that runs `BACKUP` to external storage does **not** require the [`admin` role](security-reference/authorization.html#admin-role) in the following scenarios:
+
+- Amazon S3 and Google Cloud Storage using `SPECIFIED` credentials. Azure Storage is always `SPECIFIED` by default.
+- [Userfile](use-userfile-for-bulk-operations.html)
+
+{% include_cached new-in.html version="v22.2" %} You can grant a user the `EXTERNALIOIMPLICITACCESS` [system privilege](security-reference/authorization.html#supported-privileges). This privilege allows a user without the `admin` role to interact with a cloud storage resource using `IMPLICIT` authentication.
+
+The user that runs `BACKUP` to external storage **does** require the [`admin` role](security-reference/authorization.html#admin-role) in the following scenarios:
```
Comment (adityamaru) [inline _includes/v22.2/backups/destination-privileges.md:8]: Maybe we can rephrase or restructure this sentence because there is now no ExternalStorage to which *only* an admin can backup, import, restore etc. If the user has `EXTERNALIOIMPLICITACCESS` even the custom endpoint and nodelocal are fair game for the non-admin user. 

We should probably structure it as:

`EXTERNALIOIMPLICITACCESS` is required for the following scenarios:

No special privilege is required in the following scenarios:

File: _includes/v22.2/backups/updated-backup-privileges.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{{site.data.alerts.callout_info}}
+From v22.2, CockroachDB supports a backup privilege model that provides finer control over a user's privilege to take backups. There is continued support for the [existing privilege model](#existing-required-privileges) in v22.2. However, this will be removed in v23.1.
```
Comment (adityamaru) [inline _includes/v22.2/backups/updated-backup-privileges.md:2]: We should probably not commit to removing in 23.1 cc: @livlobo. We can be a little vague and say removed in a future release?

File: _includes/v22.2/backups/updated-backup-privileges.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+{{site.data.alerts.end}}
+
+{% include_cached new-in.html version="v22.2" %} You can grant the `BACKUP` privilege to a user or role depending on the type of backup:
+
+Backup | Privilege
+-------+-----------
+Cluster | Grant a user the `BACKUP` privilege at the system level.
+Database | Grant a user the `BACKUP` privilege at the database level.
```
Comment (adityamaru) [inline _includes/v22.2/backups/updated-backup-privileges.md:10]: nit: for clarity maybe we can say "Grant a user the `BACKUP` privilege on the target database". Similarly for table since `GRANT` is executed on a database/table.

File: _includes/v22.2/backups/updated-backup-privileges.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+Table | Grant a user the `BACKUP` privilege at the table level. This includes any schemas and types associated with a table.
```
Comment (adityamaru) [inline _includes/v22.2/backups/updated-backup-privileges.md:11]: nit: instead of this includes maybe we can explicitly say "this gives the user the privilege to backup the schema and all user defined types that are associated with the table"?

File: _includes/v22.2/backups/updated-backup-privileges.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+There is no inheritance with the listed privileges. For example, if a user is granted database-level backup privileges, this does not give the user the privilege to back up a table. Furthermore, if you run an [`ALTER DEFAULT PRIVILEGES`](alter-default-privileges.html) statement to grant a user the `BACKUP` privilege, this will only apply to newly created tables.
```
Comment (adityamaru) [inline _includes/v22.2/backups/updated-backup-privileges.md:13]: What do you think about:

The listed privileges do not cascade to objects lower in the schema tree. For example, if a user is granted database-level backup privileges, this does not give the user the privilege to back up a table.

If the user wants the `BACKUP` privilege on a database to apply to all newly created tables in that database then they can leverage `DEFAULT PRIVILEGES`.

File: v22.2/import.md [hunk 6]
```diff
@@ -35,13 +35,23 @@ Optimize import operations in your applications by following our [Import Perform
...
 
-{% include {{ page.version.version }}/misc/source-privileges.md %}
+The user that runs `IMPORT` to external storage does **not** require the `ADMIN` role in the following scenarios:
+
+- Amazon S3 and Google Cloud Storage using `SPECIFIED` credentials. Azure Storage is always `SPECIFIED` by default.
+- [Userfile](use-userfile-for-bulk-operations.html)
+
+{% include {{ page.version.version }}/misc/external-io-privilege.md %}
+
+The user that runs `IMPORT` to external storage **does** require the [`admin` role](security-reference/authorization.html#admin-role) in the following scenarios:
```
Comment (adityamaru) [inline v22.2/import.md:51]: Same as the comment in `destination-privileges.md` I don't know if we want to give the user the impression that there are still operations that only admin can run, because thats not the case anymore. `EXTERNALIOIMPLICITACCESS` governs who can and cannot run against an ExternalStorage.

File: v22.2/import-into.md [hunk 7]
```diff
@@ -33,16 +33,17 @@ The user must have the `INSERT` and `DROP` [privileges](security-reference/autho
...
+- Amazon S3 and Google Cloud Storage using `SPECIFIED` credentials. Azure Storage is always `SPECIFIED` by default.
 - [Userfile](use-userfile-for-bulk-operations.html)
 
-The source file URL _does_ require the `ADMIN` role in the following scenarios:
+{% include {{ page.version.version }}/misc/external-io-privilege.md %}
 
-- S3 or GS using `IMPLICIT` credentials
-- Use of a [custom endpoint](https://docs.aws.amazon.com/sdk-for-go/api/aws/endpoints/) on S3
-- [Nodelocal](cockroach-nodelocal-upload.html), [HTTP](use-a-local-file-server-for-bulk-operations.html), or [HTTPS](use-a-local-file-server-for-bulk-operations.html)
+The user that runs `IMPORT INTO` to external storage **does** require the [`admin` role](security-reference/authorization.html#admin-role) in the following scenarios:
```
Comment (adityamaru) [inline v22.2/import-into.md:43]: Same as the comment in destination-privileges.md I don't know if we want to give the user the impression that there are still operations that only admin can run, because thats not the case anymore. EXTERNALIOIMPLICITACCESS governs who can and cannot run against an ExternalStorage.

File: v22.2/export.md [hunk 8]
```diff
@@ -34,6 +34,20 @@ The `EXPORT` statement cannot be used within a [transaction](transactions.html).
...
+### Destination privileges
+
+ The user that runs `EXPORT` to external storage does **not** require the `ADMIN` role in the following scenarios:
+
+- Amazon S3 and Google Cloud Storage using `SPECIFIED` credentials. Azure Storage is always `SPECIFIED` by default.
+- [Userfile](use-userfile-for-bulk-operations.html)
+
+{% include {{ page.version.version }}/misc/external-io-privilege.md %}
+
+The user that runs `EXPORT` to external storage **does** require the [`admin` role](security-reference/authorization.html#admin-role) in the following scenarios:
```
Comment (adityamaru) [inline v22.2/export.md:46]: Same as the comment in destination-privileges.md I don't know if we want to give the user the impression that there are still operations that only admin can run, because thats not the case anymore. EXTERNALIOIMPLICITACCESS governs who can and cannot run against an ExternalStorage.

File: v22.2/restore.md [hunk 9]
```diff
@@ -28,12 +28,44 @@ You can restore:
 
 ## Required privileges
 
+{{site.data.alerts.callout_info}}
```
Comment (adityamaru) [inline v22.2/restore.md:31]: All the comments I made for backup apply to restore too (sorry I'm lazy 😋)

File: v22.2/show-backup.md [hunk 10]
```diff
@@ -19,7 +19,19 @@ For guidance on the syntax for `SHOW BACKUP FROM`, see the [examples](#examples)
...
 
-{% include {{ page.version.version }}/misc/non-http-source-privileges.md %}
+The user that runs `SHOW BACKUP` to external storage does **not** require the [`admin` role](security-reference/authorization.html#admin-role) in the following scenarios:
+
+- Amazon S3 and Google Cloud Storage using `SPECIFIED` credentials. Azure Storage is always `SPECIFIED` by default.
+- [Userfile](use-userfile-for-bulk-operations.html)
+
+{% include {{ page.version.version }}/misc/external-io-privilege.md %}
+
+The user that runs `SHOW BACKUP` to external storage **does** require the [`admin` role](security-reference/authorization.html#admin-role) in the following scenarios:
```
Comment (adityamaru) [inline v22.2/show-backup.md:29]: Same as the comment in destination-privileges.md I don't know if we want to give the user the impression that there are still operations that only admin can run, because thats not the case anymore. EXTERNALIOIMPLICITACCESS governs who can and cannot run against an ExternalStorage.

File: v22.2/restore.md [hunk 11]
```diff
@@ -28,12 +28,44 @@ You can restore:
...
 
-{% include {{ page.version.version }}/misc/non-http-source-privileges.md %}
+The user that runs `RESTORE` from cloud storage does **not** require the [`admin` role](security-reference/authorization.html#admin-role) in the following scenarios:
+
+- Amazon S3 and Google Cloud Storage using `SPECIFIED` credentials. Azure Storage is always `SPECIFIED` by default.
+- [Userfile](use-userfile-for-bulk-operations.html)
+
+{% include {{ page.version.version }}/misc/external-io-privilege.md %}
+
+The user that runs `RESTORE` from cloud storage **does** require the [`admin` role](security-reference/authorization.html#admin-role) in the following scenarios:
```
Comment (adityamaru) [inline v22.2/restore.md:63]: Same as the comment in destination-privileges.md I don't know if we want to give the user the impression that there are still operations that only admin can run, because thats not the case anymore. EXTERNALIOIMPLICITACCESS governs who can and cannot run against an ExternalStorage.

File: _includes/v22.2/backups/destination-privileges.md (hunk 1 above)
Comment (kathancox) [reply _includes/v22.2/backups/destination-privileges.md:8]: @adityamaru OK, let me check that I understand your proposed structure... Like the following?:

EXTERNALIOIMPLICITACCESS is required for the following scenarios:

- A user without the `admin` role to interact with a cloud storage resource using `IMPLICIT` authentication.

No special privilege is required in the following scenarios:

- Amazon S3 and Google Cloud Storage using `SPECIFIED` credentials. Azure Storage is always `SPECIFIED` by default.
- [Userfile](use-userfile-for-bulk-operations.html)
- Use of a [custom endpoint](https://docs.aws.amazon.com/sdk-for-go/api/aws/endpoints/) on S3.
- [Nodelocal](cockroach-nodelocal-upload.html)

Comment (adityamaru) [reply _includes/v22.2/backups/destination-privileges.md:8]: EXTERNALIOIMPLICITACCESS or admin is required for the following scenarios:

- A user without the admin role to interact with a cloud storage resource using IMPLICIT authentication.
- Use of a [custom endpoint](https://docs.aws.amazon.com/sdk-for-go/api/aws/endpoints/) on S3.
- Nodelocal
- HTTP

No special privilege is required for:
- Amazon S3 and Google Cloud Storage using SPECIFIED credentials. 
- Azure Storage is always SPECIFIED by default.
- [Userfile](https://github.com/cockroachdb/docs/pull/use-userfile-for-bulk-operations.html)
- External Connections (external://) since they have their own privileges.

Comment (kathancox) [reply _includes/v22.2/backups/destination-privileges.md:8]: Thanks, will make the changes.

Comment (kathancox) [reply _includes/v22.2/backups/destination-privileges.md:8]: I will add in external connections with its PR. As this PR will merge first, there will be no documentation on that until the ext. connections work is complete.

File: _includes/v22.2/backups/updated-backup-privileges.md [hunk 12]
```diff
@@ -0,0 +1,17 @@
...
+
+Backup | Privilege
+-------+-----------
+Cluster | Grant a user the system level `BACKUP` privilege.
+Database | Grant a user the `BACKUP` privilege on the target database.
+Table | Grant a user the `BACKUP` privilege at the table level. This gives the user the privilege to back up the schema and all user-defined types that are associated with the table.
+
+The listed privileges do not cascade to objects lower in the schema tree. For example, if you are granted database-level `BACKUP` privileges, this does not give you the privilege to back up a table. If you need the `BACKUP` privilege on a database to apply to all newly created tables in that database, use [`DEFAULT PRIVILEGES`](security-reference/authorization.html#default-privileges).
+
+Members of the [`admin` role](security-reference/authorization.html#admin-role) can run all levels of `BACKUP` without the need to grant a specific `BACKUP` privilege.
```
Comment (livlobo) [inline _includes/v22.2/backups/updated-backup-privileges.md:15]: I am wondering if we should separate out the mentions of ... "you can also use admin... " into a separate section -- with stern warnings around the drawback of using admin? And steer them towards using the finer grained privileges via "Recommended privilege model" or something like that.

**Review by livlobo — COMMENTED** [review]
Overall comment - Looking the create schedule for backup, I found it a bit hard to follow because there were so many sections about privileges... 
- Required privileges
- Existing required privileges (if I am scanning the page, I don't know which one to read)
- Destination privileges
- Destination permissions (how is this different from privileges?)

Idea:
We could have a single line explaining what that section is for (i.e. destination privileges.. if you are in backup, destination privileges refer to what access you need on the destination where you are saving the backups.. )

**Comment by kathancox** [comment]
> Overall comment - Looking the create schedule for backup, I found it a bit hard to follow because there were so many sections about privileges...
> 
> * Required privileges
> * Existing required privileges (if I am scanning the page, I don't know which one to read)
> * Destination privileges
> * Destination permissions (how is this different from privileges?)
> 
> Idea: We could have a single line explaining what that section is for (i.e. destination privileges.. if you are in backup, destination privileges refer to what access you need on the destination where you are saving the backups.. )

I'll take a look and make some changes around the recommendations and blending the Existing privileges header. I was erring on the side of keeping these distinct so it was obvious to the user what the new privilege model is vs. previous.

Re Destination privileges vs. permissions. They are different since the privileges are set in CockroachDB and permissions are the _permissions_ required at the storage location. I think the problem is that the sections need renaming.

**Comment by kathancox** [comment]
@livlobo @adityamaru I have made some changes based on the feedback. 

- I kept the "existing privileges" header in place for now. 
- I dropped the permissions header and content from this section. The reason being is that we have a large and detailed Storage Permissions section on the cloud storage page, which covers this information. So, I have linked to that. Since those need to be set in cloud storage, which differs from the CRDB privileges that might be a better home. (We also mention the object locking in the considerations section on the `BACKUP` page, so no need to dupe really.)
- I've taken the opportunity to add a couple of recommendations to the new privilege model section.

Let me know if you would prefer any further changes! Thanks for the feedback.

**Review by adityamaru — APPROVED** [review]
Looks good to me!!

File: v22.2/restore.md [hunk 13]
```diff
@@ -28,12 +28,47 @@ You can restore:
...
+
+{% include_cached new-in.html version="v22.2" %} You can grant the `RESTORE` privilege to a user or role depending on the type of restore required:
+
+Restore | Privilege
+-------+-----------
+Cluster | Grant a user the system level `RESTORE` privilege.
+Database | Grant a user the system level `RESTORE` privilege to restore databases onto the cluster.
+Table | Grant a user the database level `RESTORE` privilege to restore schema objects into the database.
+
+The listed privileges do not cascade to objects lower in the schema tree. For example, if you are granted system-level restore privileges, this does not give you the privilege to restore a table. If you need the `RESTORE` privilege on a database to apply to all newly created tables in that database, use [`DEFAULT PRIVILEGES`](security-reference/authorization.html#default-privileges).
```
Comment (livlobo) [inline v22.2/restore.md:45]: Would they know how to use Default privileges.. that links to a very generic page. Would adding something like --> Add BACKUP to the user/role's default privileges make sense?

File: v22.2/create-schedule-for-backup.md [hunk 14]
```diff
@@ -17,9 +17,17 @@ To use the other backup features, you need an [Enterprise license](enterprise-li
 
 ## Required privileges
 
+{% include {{ page.version.version }}/backups/updated-backup-privileges.md %}
+
+### Existing required privileges
+
 - Only members of the [`admin` role](security-reference/authorization.html#default-roles) can run `CREATE SCHEDULE FOR BACKUP`. By default, the `root` user belongs to the `admin` role.
```
Comment (livlobo) [inline v22.2/create-schedule-for-backup.md:24]: This is no longer true, with the new backup privilege model in 22,2, right? 

(Or is this section meant to indicate the "current privilege model" for 22.1 and older?)

File: _includes/v22.2/backups/updated-backup-privileges.md [hunk 15]
```diff
@@ -0,0 +1,19 @@
...
+
+Backup | Privilege
+-------+-----------
+Cluster | Grant a user the system level `BACKUP` privilege.
+Database | Grant a user the `BACKUP` privilege on the target database.
+Table | Grant a user the `BACKUP` privilege at the table level. This gives the user the privilege to back up the schema and all user-defined types that are associated with the table.
+
+The listed privileges do not cascade to objects lower in the schema tree. For example, if you are granted database-level `BACKUP` privileges, this does not give you the privilege to back up a table. If you need the `BACKUP` privilege on a database to apply to all newly created tables in that database, use [`DEFAULT PRIVILEGES`](security-reference/authorization.html#default-privileges).
+
+Members of the [`admin` role](security-reference/authorization.html#admin-role) can run all levels of `BACKUP` without the need to grant a specific `BACKUP` privilege. However, we recommend using the `BACKUP` privilege model, which provides stronger access control.
```
Comment (livlobo) [inline _includes/v22.2/backups/updated-backup-privileges.md:17]: > However, we recommend using the `BACKUP` privilege model,

Would be helpful to elaborate a bit on how they can use the privilege model:
We recommend using the BACKUP privilege model to create users/roles and grant them BACKUP privileges as needed.

**Review by livlobo — APPROVED** [review]

File: v22.2/create-schedule-for-backup.md (hunk 14 above)
Comment (kathancox) [reply v22.2/create-schedule-for-backup.md:24]: It is the existing privilege model yes. I have added some preface sentences to make this clear on the support and such to this section. Since it is supported in v22.2 and earlier, I have not changed to "current" because I think we really what the fine-grained access to be "current" if that makes sense.

File: _includes/v22.2/backups/updated-backup-privileges.md (hunk 15 above)
Comment (kathancox) [reply _includes/v22.2/backups/updated-backup-privileges.md:17]: Yes, good point, done.

**Comment by kathancox** [comment]
@livlobo following from the PC, read the additional note [here](https://github.com/cockroachdb/docs/pull/15043/commits/1f90ad923aff12115ada76a3e21cf9f2771e80ab). Lmk if you need any changes.

File: _includes/v22.2/backups/destination-privileges.md [hunk 16]
```diff
@@ -0,0 +1,14 @@
+{% include_cached new-in.html version="v22.2" %} You can grant a user the `EXTERNALIOIMPLICITACCESS` [system privilege](security-reference/authorization.html#supported-privileges).
+
+`EXTERNALIOIMPLICITACCESS` or [`admin`](security-reference/authorization.html#admin-role) is required for the following scenarios:
```
Comment (andf-crl) [inline _includes/v22.2/backups/destination-privileges.md:3]: Nit: let's label these two requirements:

> Either the `EXTERNALIOIMPLICITACCESS` system privilege or the [`admin` role](security-reference/authorization.html#admin-role) are required for the following scenarios:

File: _includes/v22.2/backups/destination-privileges.md [hunk 17]
```diff
@@ -0,0 +1,14 @@
...
+
+- To interact with a cloud storage resource using [`IMPLICIT` authentication](use-cloud-storage-for-bulk-operations.html#authentication).
+- Use of a [custom endpoint](https://docs.aws.amazon.com/sdk-for-go/api/aws/endpoints/) on S3.
+- [Nodelocal](cockroach-nodelocal-upload.html)
```
Comment (andf-crl) [inline _includes/v22.2/backups/destination-privileges.md:7]: nit: We could structure these three with similar (parallel) structure for a bit more comprehensibility:

> - Interacting with a cloud storage resource using [`IMPLICIT` authentication](use-cloud-storage-for-bulk-operations.html#authentication).
> - Using a [custom endpoint](https://docs.aws.amazon.com/sdk-for-go/api/aws/endpoints/) on Amazon S3
> - Using the [`cockroach nodelocal upload`](cockroach-nodelocal-upload.html) command.

File: _includes/v22.2/backups/destination-privileges.md [hunk 18]
```diff
@@ -0,0 +1,14 @@
...
+
+No special privilege is required for: 
+
+- Amazon S3 and Google Cloud Storage using `SPECIFIED` credentials. Azure Storage is always `SPECIFIED` by default.
+- [Userfile](use-userfile-for-bulk-operations.html)
```
Comment (andf-crl) [inline _includes/v22.2/backups/destination-privileges.md:12]: nit: similarly, following the parallelization above:

> - Interacting with an Amazon S3 or Google Cloud Storage resource using `SPECIFIED` credentials. Azure Storage is always `SPECIFIED` by default.
> - Using any of the [`cockroach userfile` commands](use-userfile-for-bulk-operations.html)

File: _includes/v22.2/backups/updated-backup-privileges.md [hunk 19]
```diff
@@ -0,0 +1,19 @@
...
+
+Backup | Privilege
+-------+-----------
+Cluster | Grant a user the system level `BACKUP` privilege. For example, `GRANT SYSTEM BACKUP TO user;`.
+Database | Grant a user the `BACKUP` privilege on the target database. For example, `GRANT BACKUP ON DATABASE test_db TO user;`.
+Table | Grant a user the `BACKUP` privilege at the table level. This gives the user the privilege to back up the schema and all user-defined types that are associated with the table. For example, `GRANT BACKUP ON TABLE test_db.table TO user;`.
+
+The listed privileges do not cascade to objects lower in the schema tree. For example, if you are granted database-level `BACKUP` privileges, this does not give you the privilege to back up a table. If you need the `BACKUP` privilege on a database to apply to all newly created tables in that database, use [`DEFAULT PRIVILEGES`](security-reference/authorization.html#default-privileges). You can add `BACKUP` to the user or role's default privileges with [`ALTER DEFAULT PRIVILEGES`](alter-default-privileges.html#grant-default-privileges-to-a-specific-role).
```
Comment (andf-crl) [inline _includes/v22.2/backups/updated-backup-privileges.md]: Nice! Super helpful!

File: _includes/v22.2/backups/updated-backup-privileges.md [hunk 20]
```diff
@@ -0,0 +1,19 @@
...
+Table | Grant a user the `BACKUP` privilege at the table level. This gives the user the privilege to back up the schema and all user-defined types that are associated with the table. For example, `GRANT BACKUP ON TABLE test_db.table TO user;`.
+
+The listed privileges do not cascade to objects lower in the schema tree. For example, if you are granted database-level `BACKUP` privileges, this does not give you the privilege to back up a table. If you need the `BACKUP` privilege on a database to apply to all newly created tables in that database, use [`DEFAULT PRIVILEGES`](security-reference/authorization.html#default-privileges). You can add `BACKUP` to the user or role's default privileges with [`ALTER DEFAULT PRIVILEGES`](alter-default-privileges.html#grant-default-privileges-to-a-specific-role). 
+
+Members of the [`admin` role](security-reference/authorization.html#admin-role) can run all levels of `BACKUP` without the need to grant a specific `BACKUP` privilege. However, we recommend using the `BACKUP` privilege model to create users or roles and grant them `BACKUP` privileges as necessary for stronger access control.
```
Comment (andf-crl) [inline _includes/v22.2/backups/updated-backup-privileges.md:21]: Tricky! First usage of `BACKUP` is meant to refer to the _task_ of backing up., while remaining usages refer to the newly-introduced system privilege named `BACKUP`.

We can probably clarify this simply by removing monospace from the first instance, with a slight language tweak. Maybe something like:

> Members of the [`admin` role](security-reference/authorization.html#admin-role) can run all three types of backups (cluster, database, and table) without needing to explicitly grant a specific `BACKUP` privilege. However ...

File: v22.2/backup.md [hunk 21]
```diff
@@ -45,14 +45,20 @@ To view the contents of an backup created with the `BACKUP` statement, use [`SHO
 
 ## Required privileges
 
+{% include {{ page.version.version }}/backups/updated-backup-privileges.md %}
+
+### Existing required privileges
```
Comment (andf-crl) [inline v22.2/backup.md:50]: Hmm, this is where it'd be great to have a name of the new priv model from marketing / product. "Required privileges" is actually "now and future required privs" while "Existing required privileges" is actually "now and past required privs". This is very tricky!

APIs get to go `v2`, `v3`. CRDB gets to go `v22.1`, `v22.2`. But this feature v2.0 doesn't have a name other than (`new` && `delivered with CRDB v22.2`), so we have to get clever ourselves:

Here's some brainstorming:

> ## Legacy privileges - CockroachDB v22.2 and previous
> ## Granular privileges - CockroachDB v22.2 and later

Or maybe

> ## Required privileges using the legacy privilege model
>
> The following privileges are required to use `BACKUP` when using the legacy privilege model used in CockroachDB up to and including v22.2. Support for the legacy privilege model will be removed in a future release:
> ...
> ## Required privileges using the granular privilege model
> The following privileges are required to use `BACKUP` when using the granular privilege model introduced in CockroachDB v22.2. The granular privilege model will be the sole supported privilege model (replacing the [legacy privilege model]) in a future release:
> ...

Just brainstorming!! This is tricky and I'm not sure how to solve.

File: v22.2/backup.md [hunk 22]
```diff
@@ -45,14 +45,20 @@ To view the contents of an backup created with the `BACKUP` statement, use [`SHO
...
+
 - [Full cluster backups](take-full-and-incremental-backups.html#full-backups) can only be run by members of the [`admin` role](security-reference/authorization.html#admin-role). By default, the `root` user belongs to the `admin` role.
 - For all other backups, the user must have [read access](security-reference/authorization.html#managing-privileges) on all objects being backed up. Database backups require `CONNECT` privileges, and table backups require `SELECT` privileges. Backups of user-defined schemas, or backups containing user-defined types, require `USAGE` privileges.
-- `BACKUP` requires full read and write permissions to its target destination.
-- `BACKUP` does **not** require delete or overwrite permissions to its target destination. This allows `BACKUP` to write to cloud storage buckets that have [object locking](use-cloud-storage-for-bulk-operations.html#object-locking) configured. We recommend enabling object locking in cloud storage buckets to protect the validity of a backup.
+
+See the [previous section](#required-privileges) for the updated privilege model.
```
Comment (andf-crl) [inline v22.2/backup.md:57]: nit: this "seealso" is perfect here, but suggest using instructive link title text. Also a good chance to drive home that it won't be around forever.

> See [required legacy privileges] for the privileges required to use `BACKUP` if using the legacy privilege model, which will be removed in a future version.

**Review by andf-crl — APPROVED** [review]
LGTM with some small language nits and some brainstorming for potential "old way" vs "new way" approach. Tricky one indeed!

I left comments on some files that apply to the other like-files in like-fashion. Like comments to `_includes/v22.2/backups/destination-privileges.md` apply to `v22.2/restore.md`, etc.

Feel free to ping me if you want to chat more or brainstorm. Other than nits, this LGTM and is some really solid work!

File: v22.2/backup.md (hunk 21 above)
Comment (kathancox) [reply v22.2/backup.md:50]: So a per Andrew and mine's discussion on Slack, we opted for the new, fine-grained privilege model, being simply ## Required Privileges. Since this is what we now recommend, let's not draw a distinction between the two. We want this model to be our main and only privilege model in the future. 

"Legacy" is exactly the word we need for the existing model and provides the distinction we need. Going to mark this up and add it as another H2 header, so it doesn't "fall under" the main Required Privileges.
